src/Initializer/CellLocalMatrices.cpp

```diff
@@ -100,9 +100,11 @@ void seissol::initializers::initializeCellLocalMatrices( MeshReader const&
     {
 #endif
     real ATData[tensor::star::size(0)];
+    real AT_tildeData[tensor::star::size(0)];
     real BTData[tensor::star::size(1)];
     real CTData[tensor::star::size(2)];
     auto AT = init::star::view<0>::create(ATData);
+    auto AT_tilde = init::star::view<0>::create(AT_tildeData);
```

Comment:
why the underscore naming? What is actually AT_tilde? WHat does it mean?

Reply:
AT_tilde has the same structure as the matrix AT but with the material parameters rotated to the face aligned coordinate system. The underscore will be removed.